packages/eth2.0-utils/src/crypto.ts

```diff
@@ -0,0 +1,21 @@
+import {bytes} from "@chainsafe/eth2.0-types";
+// @ts-ignore
+import sha256, {init, update, digest} from "@chainsafe/as-sha256";
+
+export function hash(data: bytes): bytes {
+  return Buffer.from(sha256(data));
+}
+
+export class Hash {
```

Comment:
this is misleading, shouldn't exist. (It should be added to as-sha256 once available)

The builder interface is currently a singleton, so creating multiple Hash objects gives the wrong impression.

```
const h0 = new Hash();
h0.update(data);
// ...
const h1 = new Hash(); // overwrites h0 state

Reply:
yeah agreed, I've added this because if I do not, I cannot integrate it into ssz. Any idea how to organize it. Wait with this PR until we add this to as-sha256?